<launch>
    <arg name="training_dir" default="/media/Data/datasets/my_classifier/my_training_data_3D" />
    <arg name="trained_dir" default="/media/Data/datasets/my_classifier/training_data_3D" />
    <arg name="test_dir" default="/media/Data/datasets/my_classifier/my_test_data_3D" />
    <arg name="force_retrain" default="true" />
   	<arg name="features_3d" default="8" />
    <arg name="features_2d" default="0" />

	<!-- classifier service -->
	<node pkg="generic_3d_object_classifier" type="generic_3d_object_classifier_node" name="generic_3d_object_classifier_node" output="screen" >
		<param name="training_dir" value="$(arg training_dir)" type="string"/>
		<param name="trained_dir" value="$(arg trained_dir)" type="string"/>
		<param name="test_dir" value="$(arg test_dir)" type="string"/>
		<param name="features_3d" value="$(arg features_3d)" type="int"/>
		<param name="features_2d" value="$(arg features_2d)" type="int"/>
		<param name="force_retrain" value="$(arg force_retrain)" type="bool"/>
	</node>

</launch>
